modules/nw-aws-replacing-clb-with-nlb.adoc

```diff
@@ -36,14 +40,19 @@ spec:
           type: NLB
     type: LoadBalancerService
 ----
-+ 
-If your default Ingress Controller has other customizations, ensure that you modify the file accordingly. 
++
+If your default Ingress Controller has other customizations, ensure that you modify the file accordingly.
++
+[TIP]
+====
+If your Ingress Controller has no other customizations and you are only updating the load balancer type, consider following the procedure detailed in "Switching the Ingress Controller from using a Classic Load Balancer to a Network Load Balancer".
```

Comment:
I _think_ there should be another comma after customizations but not 100%
```suggestion
If your Ingress Controller has no other customizations, and you are only updating the load balancer type, consider following the procedure detailed in "Switching the Ingress Controller from using a Classic Load Balancer to a Network Load Balancer".
```

Reply:
They are closely related so I am going to keep it as is!